### ISSUE TITLE
Prohibit "use vars"

### ISSUE THREAD

**Issue by petdance** [Perl-Critic/Perl-Critic#874]
`use vars` has been discouraged since the introduction of `our` in Perl 5.6.0.

Add a policy that prohibits it.

**Comment by reneeb** [comment]
Variables::ProhibitPackageVars already checks for that. Do you want a separate policy?

**Comment by petdance** [comment]
No, but now I'm wondering why I wasn't getting my code detecting it.  I'll have to try to recreate it.
